src/index.ts

```diff
@@ -126,6 +126,9 @@ const utilityPatterns: Record<string, string | [string, string | string[]]> = {
   // Flexbox
   basis: "flexBasis",
   flex: "flex",
+  gap: "gap",
+  "gap-x": ["gap", "columnGap"],
+  "gap-y": ["gap", "rowGap"],
```

Comment:
Wasn't totally sure what the array syntax implied here - just tried to mirror existing rules like `ml: ["margin", "marginLeft"]`